src/containers/ProjectCard/ProjectCard.js

```diff
@@ -1,10 +1,23 @@
 import React from 'react';
 
-export const ProjectCard = () => {
+export const ProjectCard = (props) => {
+  
+    const palettes = props.palettes.map(palette => {
+      return <div className='card-palette'>
+        <span>{palette.name}</span>
+        <div style={{ backgroundColor: `#${palette.color1}` }} className='palette-color-div'></div>
```

Comment:
I think we are already saving the # when the color is set in the setRandomColors() function on palette. We could also refactor this to be a for loop since we know there will always be 5 colors.

Reply:
ya, I agree. Wasn't sure if we wanted to make these Components, I just wanted to test we were getting the right info. But we should refactor either way.